messaginginapp/src/main/java/io/customer/messaginginapp/gist/data/sse/HeartbeatTimerEvent.kt

```diff
@@ -0,0 +1,12 @@
+package io.customer.messaginginapp.gist.data.sse
+
+/**
+ * Events emitted by HeartbeatTimer when timer state changes.
+ */
+internal sealed class HeartbeatTimerEvent {
```

Comment:
are there more events we are expecting here? if not then this can be simple object